android-components/components/compose/awesomebar/src/main/java/mozilla/components/compose/browser/awesomebar/internal/Suggestions.kt

```diff
@@ -57,6 +66,26 @@ internal fun Suggestions(
             }
         }
     }
+
+    // The latest list of suggestions with which this composable was recomposed.
+    val currentSuggestions by rememberUpdatedState(suggestions)
+
+    // The latest `onVisibilityStateUpdated` callback with which this composable was composed.
```

Comment:
nit: I do appreciate these comments because `rememberUpdatedState` is new to me but I think as Compose familiarity grows the behavior of these properties should be self-evident enough that we don't need the comments.

Reply:
Agreed; removed both comments!

I was inspired to add them after reading [this helpful explanation](https://jetc.dev/slack/2021-08-28-why-need-rememberUpdateState.html) of why we need to use `rememberUpdatedState` with `LaunchedEffect`—and wanted to share it in case you find it helpful, too!